voluncheer/voluncheer/urls.py

```diff
@@ -18,28 +18,14 @@
 from django.urls import path
 from django.contrib.auth import views as auth_views
 
-from profiles.views.home import SignUpView
-from profiles.views.organizations import OrganizationSignUpView
-from profiles.views.volunteers import VolunteerSignUpView
 
 urlpatterns = [
-    path("__debug__/", include("debug_toolbar.urls")),
-    path("admin/", admin.site.urls),
+    path("", include("django.contrib.auth.urls")),
     path("", include("profiles.urls")),
-    path("accounts/", include("django.contrib.auth.urls")),
```

Comment:
Using the `accounts` prefix is what the django documentation had as an example. They don't really go into detail as to why but I assume to avoid overload/conflict. As long as we OK not using those url patterns, this is fine.

Reply:
Just to be safe, I put the `accounts` back to the URLConf that leads to authentication URLConf. But it shouldn't be necessary for any of the signup views--this URLConf is simply for login/logout/password reset/password change.